src/main/kotlin/com/openlattice/mechanic/upgrades/CreateAndPopulateOrganizationDatabaseTable.kt

```diff
@@ -0,0 +1,116 @@
+package com.openlattice.mechanic.upgrades
+
+import com.openlattice.assembler.AssemblerConfiguration
+import com.openlattice.assembler.AssemblerConnectionManager
+import com.openlattice.assembler.PostgresDatabases
+import com.openlattice.hazelcast.HazelcastMap
+import com.openlattice.mechanic.Toolbox
+import com.openlattice.postgres.PostgresColumn.*
+import com.openlattice.postgres.PostgresTable.ORGANIZATION_DATABASES
+import com.openlattice.postgres.streams.BasePostgresIterable
+import com.openlattice.postgres.streams.StatementHolderSupplier
+import com.zaxxer.hikari.HikariDataSource
+import org.slf4j.LoggerFactory
+import java.util.*
+
+class CreateAndPopulateOrganizationDatabaseTable(
+        private val toolbox: Toolbox,
+        private val assemblerConfiguration: AssemblerConfiguration
+) : Upgrade {
+
+    companion object {
+        private val logger = LoggerFactory.getLogger(CreateAndPopulateOrganizationDatabaseTable::class.java)
+    }
+
+    override fun upgrade(): Boolean {
+        createTable()
+
+        createTableEntries()
+
+        return true
+    }
+
+    override fun getSupportedVersion(): Long {
+        return Version.V2020_09_15.value
+    }
+
+    private fun createTable() {
+        val createSql = ORGANIZATION_DATABASES.createTableQuery()
```

Comment:
you'll have to manually create table otherwise mechanic will fail to start

Reply:
like if it tries to load the mapstore? would that get triggered?